11:19:29 | PIX_RPA_MAIN | INFO | üöÄ Inicializando PIX RPA Process
11:19:29 | PIX_RPA_MAIN | INFO | ‚úì Gestores principales inicializados
11:19:29 | PIX_RPA_MAIN | INFO | ‚úì Gestores opcionales inicializados
11:19:29 | PIX_RPA_MAIN | INFO | üöÄ Iniciando proceso PIX RPA
11:19:29 | PIX_RPA_MAIN | INFO | üîç Validando entorno PIX RPA...
11:19:29 | PIX_RPA_MAIN | INFO |   ‚úì Directorio verificado: data/
11:19:29 | PIX_RPA_MAIN | INFO |   ‚úì Directorio verificado: reports/
11:19:29 | PIX_RPA_MAIN | INFO |   ‚úì Directorio verificado: logs/
11:19:29 | PIX_RPA_MAIN | INFO |   ‚úì Directorio verificado: evidencias/
11:19:29 | PIX_RPA_MAIN | INFO |   ‚úì Directorio verificado: config/
11:19:29 | PIX_RPA_MAIN | INFO |   ‚úì Dependencias cr√≠ticas verificadas
11:19:29 | PIX_RPA_MAIN | INFO | ‚úÖ Validaci√≥n de entorno completada
11:19:29 | PIX_RPA_MAIN | INFO | üöÄ PASO 1: Consumo de API p√∫blica
11:19:29 | PIX_RPA_MAIN | INFO | ‚úÖ Productos obtenidos: 20
11:19:29 | PIX_RPA_MAIN | INFO | ‚òÅÔ∏è Subida de JSON a OneDrive deshabilitada o no configurada.
11:19:29 | PIX_RPA_MAIN | INFO | üöÄ PASO 2: Almacenamiento en base de datos
11:19:29 | PIX_RPA_MAIN | INFO | ‚úÖ Productos insertados en BD: 0
11:19:29 | PIX_RPA_MAIN | INFO | üöÄ PASO 3: Generaci√≥n de reporte Excel
11:19:29 | PIX_RPA_MAIN | INFO | ‚úÖ Reporte Excel generado: /home/camper/Descargas/-prueba-t-cnica-RPA--master/PRUEBA-RPA-main/reports/Reporte_2025-09-10.xlsx
11:19:29 | PIX_RPA_MAIN | INFO | üöÄ PASO 4: Subida a OneDrive
11:19:29 | PIX_RPA_MAIN | INFO | ‚òÅÔ∏è OneDrive deshabilitado por configuraci√≥n
11:19:29 | PIX_RPA_MAIN | INFO | üöÄ PASO 5: Automatizaci√≥n web (formulario)
11:19:29 | WDM | INFO | ====== WebDriver manager ======
11:19:29 | WDM | INFO | Get LATEST chromedriver version for google-chrome
11:19:29 | WDM | INFO | Get LATEST chromedriver version for google-chrome
11:19:29 | WDM | INFO | Driver [/home/camper/.wdm/drivers/chromedriver/linux64/134.0.6998.165/chromedriver-linux64/THIRD_PARTY_NOTICES.chromedriver] found in cache
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de5c30>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6d40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de5ea0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6230>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de63e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de65c0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6a40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6560>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6950>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de62f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de5e40>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de60e0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de56f0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de5d20>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6c50>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/elements
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de7160>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de7370>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de7580>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de7a00>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de7520>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de7910>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de7220>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6bf0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de7040>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de5900>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de60b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de5d80>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/element
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de61d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/screenshot
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6260>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/screenshot
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de6a70>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036/screenshot
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=2, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de5510>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=1, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de66b0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036
11:20:24 | urllib3.connectionpool | WARNING | Retrying (Retry(total=0, connect=None, read=None, redirect=None, status=None)) after connection broken by 'NewConnectionError('<urllib3.connection.HTTPConnection object at 0x710fb1de64d0>: Failed to establish a new connection: [Errno 111] Connection refused')': /session/4bdd08cc97375c49cdeee2784246d036
11:20:24 | PIX_RPA_MAIN | WARNING | ‚ö†Ô∏è No se pudo enviar el formulario web
11:20:24 | PIX_RPA_MAIN | INFO | üöÄ PASO 6: Registro de evidencias
11:20:24 | PIX_RPA_MAIN | INFO | ‚úÖ Evidencias registradas en: /home/camper/Descargas/-prueba-t-cnica-RPA--master/PRUEBA-RPA-main/evidencias/evidencia_2025-09-10_11-20-24.json
11:20:24 | PIX_RPA_MAIN | INFO | ================================================================================
11:20:24 | PIX_RPA_MAIN | INFO | üéâ PROCESO PIX RPA FINALIZADO
11:20:24 | PIX_RPA_MAIN | INFO | ‚è±Ô∏è  Tiempo total: 55.40 segundos
11:20:24 | PIX_RPA_MAIN | INFO | üìä Productos procesados: 20
11:20:24 | PIX_RPA_MAIN | INFO | üìÅ Archivos generados: 2
11:20:24 | PIX_RPA_MAIN | INFO | ‚úÖ Pasos completados: 4/6
11:20:24 | PIX_RPA_MAIN | INFO | üìÇ Archivos creados:
11:20:24 | PIX_RPA_MAIN | INFO |    ‚Ä¢ /home/camper/Descargas/-prueba-t-cnica-RPA--master/PRUEBA-RPA-main/data/raw/Productos_2025-09-10.json
11:20:24 | PIX_RPA_MAIN | INFO |    ‚Ä¢ /home/camper/Descargas/-prueba-t-cnica-RPA--master/PRUEBA-RPA-main/reports/Reporte_2025-09-10.xlsx
11:20:24 | PIX_RPA_MAIN | INFO | ================================================================================
